openmls/src/ciphersuite/aead.rs

```diff
@@ -41,7 +59,7 @@ impl AeadKey {
         log::trace!("AeadKey::from_secret with {}", secret.ciphersuite);
         AeadKey {
             aead_mode: secret.ciphersuite.aead_algorithm(),
-            value: secret.value,
+            value: AeadValue(secret.as_slice().to_vec()),
```

Comment:
Avoid copying
```suggestion
value: AeadValue(secret.value),
```

Reply:
You cannot move out of an object that implements Drop. This won't compile.

Reply:
Right then like this:
```
let value = std::mem::take(&mut secret.value);
        AeadKey {
            aead_mode: secret.ciphersuite.aead_algorithm(),
            value: AeadValue(value),
        }
```